testutils/infra/device.py

```diff
@@ -330,7 +330,7 @@ def _put(device, file, local_path=".", remote_path="."):
     (scp, host, port) = _scp_prep_args(device)
 
     subprocess.check_call(
-        "%s %s %s/%s %s@%s:%s"
+        "%s -O %s %s/%s %s@%s:%s"
```

Comment:
not that this brings anything to the scp question, but I would think with that amount of params an f-string would help with readability

Reply:
hah, yeh. We have a fair few of this. Might as well move to it now!

Fixing :)

Reply:
See https://github.com/mendersoftware/integration/pull/2072/commits/3d9b69e2986ee96d5a7167bf86ab2c681bc46b82 @mzedel